src/main/java/nl/tudelft/pixelperfect/client/ServerListener.java

```diff
@@ -54,6 +55,7 @@ public void messageReceived(HostedConnection source, Message message) {
     } else if (message instanceof EventCompletedMessage) {
       EventCompletedMessage eve = (EventCompletedMessage) message;
       System.out.println("Received a completed event: " + eve.getCompletedEvent());
+      //app.getSpaceship().getLog().complete(eve.getCompletedEvent());
```

Comment:
This line needs to be uncommented (and make sure the message can be parsed to an integer)